# Parameters for the HYBRID3
hybrid3:
 kil: 1.0
 kpv: 5.0
 kdv: 2.0
task_init_point:
 pi_x: 0.6
 pi_y: 0.01
 pi_z: 0.5
task_final_point: 
 pf_x: 0.8
 pf_y: 0.01
 pf_z: 0.5
force_axis: 
 zf_s_x: 0.0
 zf_s_y: 0.0
 zf_s_z: 1.0
